components/dropzone/integration.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Integration
+page_title: DropZone Integration
+description: Integration of the Blazor DropZone with FileSelect and Upload.
+slug: dropzone-integration
+tags: telerik,blazor,dropzone,external,drag,drop,file,integration,fileselect,upload
+published: True
+position: 5
+---
+
+# DropZone Integration
```

Comment:
I think we can completely skip this article with 1-2 additional sentences in the Overview article.